lib/build-cron.js

```diff
@@ -22,8 +23,11 @@ module.exports = function buildCronJobs(cronConfigsDir, collectionName, logClean
 
   if (!cronConfigsDir || !collectionName) {
     const err = new VError('Error: \'cronConfigsDir\' and \'collectionName\' are required params.');
-    raygun.send(err);
-    throw err;
+    return Promise.resolve(raygun.send(err))
```

Comment:
Ahh okay....so final question, why are we doing promises at all?

Why not just: 
```
if (!cronConfigsDir || !collectionName) {
    const err = new VError('Error: \'cronConfigsDir\' and \'collectionName\' are required params.');
    raygun.send(err);
    log.info('raygun error sent');
    throw err;
  }
```